doc/changelog/10-standard-library/15597-exprt-bad-perm-instances.rst

```diff
@@ -0,0 +1,5 @@
+- **Changed:**
+  typeclass instances `Permutation_app'` and `Permutation_cons` are `#[export]` instead of `#[global]`
```

Comment:
```suggestion
- **Changed:**
  typeclass instances `Permutation_app'` and `Permutation_cons` to `#[export]` instead of `#[global]`
```

But this depends on if you read "Changed" too. @Zimmi48 @jfehrle  What do you think?

Reply:
Looks good to me, though I think "to" should be "to use".